client/components/pullRequests.js

```diff
@@ -4,13 +4,16 @@ import {fetchOpenPullRequests} from "../actions/index";
 
 class PullRequest extends Component {
 
+
     componentDidMount() {
         console.log("In component did mount");
         this.props.fetchOpenPullRequests();
 
     }
 
     render() {
+        var dateOptions = {year: 'numeric', month: 'short', day: 'numeric'};
```

Comment:
use 'const' here instead of 'var'
```suggestion
        const dateOptions = {year: 'numeric', month: 'short', day: 'numeric'};
```

Reply:
Done :)